nuitka/distutils/bdist_nuitka.py

```diff
@@ -105,14 +105,27 @@ def _buildPackage(self, build_lib):
             "--output-dir=%s" % output_dir,
             "--recurse-dir=%s" % self.main_package,
             "--recurse-to=%s" % self.main_package,
-            "--recurse-not-to=*.tests",
             "--show-modules",
-            "--remove-output",
-            main_filename,
+            "--remove-output"
         ]
 
+        # Process any extra options from setuptools
+        if 'nuitka' in self.distribution.command_options:
+            for option, details in self.distribution.command_options['nuitka'].items():
+                option = "--" + option.lstrip('-')
+                source, value = details
+                if value is None:
+                    command.append(option)
+                elif hasattr(value, '__iter__'):
+                    for val in value:
+                        command.append('%s=%s' % (option, val))
+                else:
+                    command.append('%s=%s' % (option, value))
+
+        command.append(main_filename)
+
         subprocess.check_call(
-            command
+            command, cwd=build_lib
```

Comment:
This seems new, probably a fix of its own right?

Reply:
Oh that's probably superfluous as there is a os.chdirearlier in the file and a restore just after this, however if the other things in this function can be fixed to not need the os.chdir this is probably safer (no chance of other threads os.chdir out from under you.... if there was ever other threads at play)